core/templates/pages/exploration-editor-page/services/change-list.service.ts

```diff
@@ -110,6 +104,12 @@ export class ChangeListService implements OnInit {
       });
   }
 
+  ngOnInit(): void {
+    this.loaderService.onLoadingMessageChange.subscribe(
+      (message: string) => this.loadingMessage = message
+    );
+  }
```

Comment:
Can you please move this also to the constructor as per the discussion in the comment above?

Reply:
done